server/swissvoice/static/css/style.css

```diff
@@ -4,6 +4,34 @@ body {
     padding: 0;
 }
 
+#bg-bubble {
+  background-image: url(../images/background/sprechblase.svg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  position: fixed;
+/* positioning*/
+  bottom: -50vh;
+  right: 0;
+  width: 50rem;
```

Comment:
is there a good reason to use "rem" here?

Reply:
Usong rem so the img is better controlable and dosen't stretch wierdly when making it responsiv